.gitignore

```diff
@@ -75,6 +75,7 @@ docs/_build/
 target/
 
 # Jupyter Notebook
+*.ipynb
```

Comment:
@alessiocastrica is it possible not to ignore `*.ipynb`? would like to keep an option to add `*.ipynb` files into this repository for providing quickly runnable examples in the future.